show-viewshed-from-point-in-scene/src/main/java/com/esri/arcgismaps/sample/showviewshedfrompointinscene/screens/MainScreen.kt

```diff
@@ -0,0 +1,287 @@
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgismaps.sample.showviewshedfrompointinscene.screens
+
+import android.app.Application
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Checkbox
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Slider
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.esri.arcgismaps.sample.sampleslib.components.SampleTopAppBar
+import com.esri.arcgismaps.sample.showviewshedfrompointinscene.components.ComposeSceneView
+import com.esri.arcgismaps.sample.showviewshedfrompointinscene.components.SceneViewModel
+
+/**
+ * Main screen layout for the sample app
+ */
+@Composable
+fun MainScreen(sampleName: String, application: Application) {
+    // create a ViewModel to handle SceneView interactions
+    val sceneViewModel = SceneViewModel(application)
+
+    Scaffold(
+        topBar = { SampleTopAppBar(title = sampleName) },
+        content = {
+            Box {
+                Column(
+                    modifier = Modifier
+                        .fillMaxSize()
+                        .padding(it)
+                ) {
+                    // composable function that wraps the SceneView
+                    ComposeSceneView(
+                        modifier = Modifier.fillMaxSize().weight(1f),
+                        sceneViewModel = sceneViewModel
+                    )
+                    // sliders
+                    HeadingSlider(sceneViewModel)
+                    PitchSlider(sceneViewModel)
+                    HorizontalAngleSlider(sceneViewModel)
+                    VerticalAngleSlider(sceneViewModel)
+                    MinimumDistanceSlider(sceneViewModel)
+                    MaximumDistanceSlider(sceneViewModel)
+                    // checkbox
+                    Row {
+                        FrustumCheckBox(sceneViewModel)
+                        AnalysisCheckBox(sceneViewModel)
+                    }
```

Comment:
Would be nice to have this defined in a `ViewshedOptionsScreen.kt` and pass the `sceneViewModel` to the compose screen component